src/pages/MeetingPage/MeetingPage.tsx

```diff
@@ -43,23 +67,18 @@ const MeetingPage = () => {
       memberPropertiesInput: [
         {
           member_id: currentMemberId,
-          property_id: categoryData?.property.find(({ name }) => name === '廣告素材')!.id,
-          value: `${
-            !adProperty
-              ? 'inbound_英鎊'
-              : adProperty && adProperty.includes('inbound_英鎊')
-              ? adProperty.slice(adProperty.indexOf('inbound_英鎊') - 1, 1).join(',') + ',inbound_英鎊'
-              : adProperty.join(',') + ',inbound_英鎊'
-          }`,
+          property_id: categoryData?.property.find(({ name }) => name === '廣告素材')?.id,
+          value: adPropertyValue,
+        },
+        {
+          member_id: currentMemberId,
+          property_id: categoryData?.property.find(({ name }) => name === '行銷活動')?.id,
+          value: marketingActivitiesPropertyValue,
         },
       ],
     },
   })
 
-  const isInsertingAdProperty = JSON.parse(settings['custom.ad_property.list'] || '{}').some(
-    (value: { behavior: string }) => value['behavior'] === 'meeting',
-  )
-
```

Comment:
I'm not sure about this. It seems like a custom requirement. Is it only for `xuemi` and `sixdigital` that needs to add `inbound_英鎊`?
If so,`custom.ad_property.list` shouldn't be removed because it seems to control this custom requirement.